app/presenters/queries/content_item_presenter.rb

```diff
@@ -34,22 +34,31 @@ def initialize(scope, params = {})
         self.order = params[:order]
         self.limit = params[:limit]
         self.offset = params[:offset]
+        self.search_query = params[:search_query]
       end
 
       def present_many
-        scope = ::Queries::GetLatest.call(self.scope)
-        scope = join_supporting_objects(scope)
-        scope = order_and_paginate(scope)
-
-        extract_fields(scope)
+        parsing_enumerator(evaluate_query(ordered_extracted_fields))
       end
```

Comment:
I find it quite difficult to read these nested method calls. I found the procedural style easier to read, but maybe that's just me? By breaking everything into one line methods, I personally struggle to build a global understanding of what this class does.